perfkitbenchmarker/cloudstack/__init__.py

```diff
@@ -1,13 +0,0 @@
-# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
Looks like this file was just deleted instead of being moved - any reason for that?